script/ghci-flags

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Computes the flags for ghcide to pass to ghci. You probably won’t be running this yourself, but rather ghcide will via configuration in hie.yaml.
```

Comment:
The approach here is similar to the approach of #373: basically just listing all the options we can think of, collected by clever use of `cabal repl -v`.

A more robust approach would probably be to do something like `cabal repl` for each component and union together the options, but that actually seems pretty challenging:

- stuff like package flags come in as two flags
- some of the flags, like `-outputdir`, probably shouldn’t be duplicated
- ordering matters for a lot of the flags
- some of them conflict with one another, like `-i` erasing any previously configured load paths
- we probably don’t want to pass the module names in as that would force `ghcide` to load everything eagerly rather than when only when you actually look at the file

Maybe someday all of this will be made unnecessary by `cabal` itself, but until that happy day, this is a pretty good solution for loading & typechecking all of our sources in `ghcide`, at the cost of requiring us to do a bit more manual work around the project’s packages & versions.